script/tool/lib/src/update_excerpts_command.dart

```diff
@@ -146,6 +146,12 @@ class UpdateExcerptsCommand extends PackageLoopingCommand {
                   case '.cpp':
                     language = 'c++';
                     break;
+                  case '.m':
+                    language = 'objectivec';
+                    break;
```

Comment:
@stuartmorgan Seems like we don't have support for header files for either language. I'm not sure there's a simple solution though.

Reply:
Good point. That would need either more special casing [here](https://github.com/flutter/packages/blob/dde5ebf5a4bad528882837eeb9fab1cf978947d8/script/tool/lib/src/update_excerpts_command.dart#L187-L191), or a more generalized solution (like a mapping from extensions to the set of allowed languages)).